File: ABC/ABC373/A/main.go
package main

import "fmt"

func main() {
	res := 0
	for i := 0; i < 12; i++ {
		var s string
		fmt.Scan(&s)
		if len(s) == i + 1 {
			res++;
		}
	}
	fmt.Println(res)
}

File: ABC/ABC373/A/main.cc
#include <iostream>
#include <string>

int main() {
  int res = 0;
  for (size_t i = 0; i < 12; ++i) {
    std::string s;
    std::cin >> s;
    if (size(s) == i + 1) ++res;
  }
  std::cout << res << std::endl;
  return 0;
}